### PR TITLE
Moved repeated code into functions.php

### PR DESCRIPTION
**Pull request by bearbin**

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -6,14 +6,10 @@
 error_reporting(E_ALL);
 
 require_once 'functions.php';
-require_once 'helpers/templater.php';
-require_once 'helpers/imagehelper.php';
+require_once 'helpers/imagehelper.php'
```
Comment (tigerw) [inline index.php]: da whoopsies with accidentally a semicolon.

**Comment by bearbin** [comment]
Fixed.

File: edit.php [hunk 2]
```diff
@@ -31,12 +26,12 @@
...
 	$Template->SetRefresh('showplugin.php?id=' . $_GET['id']);
-	return;	
+	return;
 }
 
 if (isset($_POST['DeleteConfirmed' . $_GET['id']]))
 {
-	GitHubAPI::DeleteRepositoryUpdateHook($_GET['id'], $Query['UpdateHookID']);	
+	GitHubAPI::DeleteRepositoryUpdateHook($_GET['id'], $Query['UpdateHookID']);
 	$SQLLink->query('DELETE FROM PluginData WHERE RepositoryID = %i', $_GET['id']);
```
Comment (NiLSPACE) [inline edit.php:35]: This has nothing to do with this PR, but this looks like a possible SQL injection here. I can't get really wrap my head around the way the MeekroDB class works. I got no further than [this](https://github.com/cuberite/plugin-repository/blob/bb0832c4a3fa4469811278caf9d7e90986787a93/helpers/meekrodb.php#L589)

Comment (bearbin) [inline edit.php:35]: MeekroDB should avoid injection attacks. I don't know why this project uses it over PDO, they're both about the same complexity, but it shouldn't be vulnerable to injection.

**Comment by bearbin** [comment]
Closing as woefully outdated.

**Comment by tigerw** [comment]
Sorry I never properly reviewed...

At least it's merged in spirit; MeekroDB initialisation indeed happens globally now...

**Pull request closed without merging**